### PR TITLE
DEFECT: fix password reset#549

### PR DESCRIPTION
**Pull request by Zinminhtet21801**

### CODE REVIEW

File: views/pages/passwordresetconfirmation.pug [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    $.ajax({
+                        type: 'POST',
+                        url: `/profile/password/reset/${token}`,
+                        data: {
+                            password,
+                            passwordConfirm,
+                        },
+                        success: function (response, textStatus, xhr) {
+                            $('#submit-btn').prop('disabled', false);
+                            console.log({response}, {textStatus})
```
Comment (Enubia) [inline views/pages/passwordresetconfirmation.pug:116]: please remove this log

File: views/pages/passwordresetconfirmation.pug [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                        error: function (response, textStatus, errorThrown) {
+                            $('#submit-btn').prop('disabled', false);
+                            showToast(response.responseJSON.error.msg);
+                            grecaptcha.reset();
+                        },
+                    });
+                };
+
+                window.confirmCaptchaSubmit = confirmCaptchaSubmit;
+            });
```
Comment (Enubia) [inline views/pages/passwordresetconfirmation.pug:95]: please fix indentations

**Review by Enubia — CHANGES_REQUESTED** [review]

**Comment by Zinminhtet21801** [comment]
Done.

**Review by Enubia — APPROVED** [review]

**Comment by stacysealky** [comment]
@Zinminhtet21801  @Enubia  have you guys tested this in https://donate-gifts-test.fly.dev/?

**Comment by Enubia** [comment]
> @Zinminhtet21801  @Enubia  have you guys tested this in https://donate-gifts-test.fly.dev/? 

No not yet but it's deployed

**Pull request merged**